### DISCUSSION TITLE
Small change request in documentation regarding XAMPP

### DISCUSSION THREAD

**Discussion by burgich** [snipe/snipe-it#8955] [Q&A] [answered]
Hi there,

I'm right now setting up snipe-it in a XAMPP environment and I think I found a small mistake in the corresponding documentation (https://snipe-it.readme.io/docs/xampp)

![grafik](https://user-images.githubusercontent.com/76842252/103442028-bb663900-4c52-11eb-97b1-2adb865fee40.png)

In the line <VirtualHost laravel.dev:80>, "laravel.dev" should be changed to "snipe-it.local" to match the other entries shown in this documentation.

Thank you!

**Comment by uberbrady** [comment, marked as answer]
This makes total sense to me - would you be willing to do the "suggest a change" thing on the Readme.io site? That's a lot easier for us, and we can just say "accept!"

**Reply by burgich** [reply]
Hi, thanks for your feedback.

For "suggest edits", an account at readme.io is needed. To be honest, I don't want to open up a new account just to change this few letters...
That's why I posted it here :)

**Reply by snipe** [reply]
This has been updated. In the future though, it would be m ore helpful if you used the "Suggest changes" option. I get that you don't want to create a new account for this, but we work really hard on providing a lot of stuff for people for free, so asking for help from the public for something like this shouldn't seem like so much of a burden.

Please mark this discussion as "answered".
